pkgs/misc/lguf-brightness/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, libusb, ncurses }:
```

Comment:
We should depend on `libusb1` and `ncurses5` instead to not have to specify which version in the all-packages file.